coreUtilities/build.gradle

```diff
@@ -51,6 +51,12 @@ dependencies {
     implementation group: 'io.opentelemetry', name: 'opentelemetry-exporter-otlp'
     implementation group: 'io.opentelemetry.semconv', name: 'opentelemetry-semconv'
 
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind'
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-core'
```

Comment:
These should not be here.  coreUtilities is pulled in by every single package.  For example, there's no reason for netty wire loggers or the proxy to be pulling jackson in.  Please break the transformers into their own common library.

Reply:
Done!